build.gradle

```diff
@@ -138,50 +119,24 @@ dependencies {
     implementation 'io.prometheus:simpleclient_httpserver'
     implementation 'io.prometheus:simpleclient_log4j2'
 
-    implementation("com.squareup.okio:okio:3.4.0") {
-        because('fix vulnerability in transitive dependency <com.squareup.okhttp3:okhttp>' +
-                'Note: kotlin:1.9 is transitive dependency since 3.5.0')
```

Comment:
question: Is it still true? If it is, we will have a kotlin `1.9.*` in common classpath instead of kotlin `1.8.*`. Maybe we can migrate to Kotlin `1.9.*` in the common library?